.travis.yml

```
@@ -1,21 +1,31 @@
 language: node_js
+cache: yarn
+
+os:
+  - windows
+  - linux
+  - osx
+
+node_js:
+  - 6
+  - 8
+  - 10
+  - 11
```

Let's fail early and use `node` instead of `11` or what do you think @ncoden?